perf-tests/src/perf-tests.py

```diff
@@ -43,13 +43,30 @@ def is_system_running(core_api, jobs_api):
 
 
 def check_system(core_api, jobs_api):
+    # try to access system endpoints
     print("Checking: core API and JOBS API endpoints")
     if not is_system_running(core_api, jobs_api):
         print("Fatal: tested system is not available")
         sys.exit(1)
     else:
         print("    ok")
 
+    # check the authorization token for the core API
+    print("Checking: authorization token for the core API")
+    if core_api.check_auth_token_validity():
+        print("    ok")
+    else:
+        print("Fatal: auth. token is probably outdated")
+        sys.exit(1)
+
+    # check the authorization token for the jobs API
+    print("Checking: authorization token for the jobs API")
+    if jobs_api.check_auth_token_validity():
+        print("    ok")
+    else:
+        print("Fatal: auth. token is probably outdated")
```

Comment:
This could be confusing if something else goes wrong. Jobs should report [401 with error message](https://github.com/fabric8-analytics/fabric8-analytics-jobs/blob/master/f8a_jobs/utils.py#L94), API should report [error with 401 as well](https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/auth.py#L69). A check for HTTP status code could do the trick or the error message could be adjusted. WDYT?

Reply:
... Or report directly what was received from the remote not to add additional code. Error messages should describe errors.

Reply:
Good idea. Thank you